resources/zib/zib-MultidisciplinaryTeamMeeting.Plan.xml

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-MultidisciplinaryTeamMeeting.Plan" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-MultidisciplinaryTeamMeeting.Plan" />
+  <name value="ZibMultidisciplinaryTeamMeetingPlan" />
+  <title value="zib MultidisciplinaryTeamMeeting.Plan" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="A multidisciplinary team meeting is a meeting of health care providers in which the results of examinations and treatment are discussed with the aim of formulating joint advice for the treatment of the patient. A patient can also be present. When healthcare professionals from various disciplines participate, and this is structurally organized (local, regional, sub regional or through a partnership), this is often referred to as a multi-disciplinary consultation ." />
+  <purpose value="This CarePlan resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) MultidisciplinaryTeamMeeting-v1.0(2020EN)](https://zibs.nl/wiki/MultidisciplinaryTeamMeeting-v1.0(2020EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-multidisciplinaryteammeeting-v1.0-2020EN" />
+    <uri value="https://zibs.nl/wiki/MultidisciplinaryTeamMeeting-v1.0(2020EN)" />
+    <name value="zib MultidisciplinaryTeamMeeting-v1.0(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="CarePlan" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/CarePlan" />
+  <derivation value="constraint" />
+  <differential>
```

Comment:
Should we constrain the 'subject' of the CarePlan to nl-core-patient?

Reply:
This is done in the nl-core versions of the profiles, which are the ones that are actually implemented, as the zib do not reference the patient anymore. See https://informatiestandaarden.nictiz.nl/wiki/FHIR:V1.0_FHIR_Profiling_Guidelines_R4#The_nl-core_layer